README.rst

```diff
@@ -68,3 +68,28 @@ To re-enable them:
 
   config setprop janus-gateway DisabledPlugins ""
   signal-event nethserver-janus-update
+  
+Debug problems with Admin Api
+=============================
+
+The Janus Admin Api are used to debug problems (e.g. ssl, ice, ...).
+  
+``nethserver-janus`` automatically enables Admin Api on HTTP ``localhost`` on port ``7088``.
+
+To use it:
+
+1. make an ssh tunnel:
+
+.. code:: bash
+
+  ssh IP -L 7088:localhost:7088
```

Comment:
```suggestion
1. make an ssh tunnel: ::
 
     ssh IP -L 7088:localhost:7088
 
```

Reply:
I've followed the syntax already present in other section of the same file.
At the time of writing the doc I checked the [documentation](https://www.sphinx-doc.org/en/1.5/markup/code.html): probably we can change `code` to `code-block` but the GitHub preview [shows the code](https://github.com/NethServer/nethserver-janus/blob/9a2c68865f9bf6dd6a7efaafc1f2a461f3c0591d/README.rst) in the right manner even using `code`

Reply:
I'm not against code, only fix the list structure with the right indentation. RST is really picky about it!